README.MD

```diff
@@ -0,0 +1,37 @@
+# IMF  Best Practices | Packaging & Delivery
+
+Best practices for IMF packaging and delivery.
+
+Current working version can be viewed [here](https://docs.google.com/document/d/1g3E-b52SY98tcREANs1Ud-sQsmRbOM-BMYGpV9UgnIQ).
+
+## Contributing
+Tracking comments in Google Docs or Word can be tedious as being word processors, they do not offer the relevant functionality.  If you have an issue with something within the document or wish to contribute to it, please adhere to the process outlined as follows;
+
+ 1. Create a [new issue](https://github.com/imfug/002-pkg-and-delivery/issues) within this repository clearly stating the
+    issue/contribution as succinct as possible in the title.
+ 2. Elaborate on it in the issue body.
+ 3. Back in the document, comment with the issue ID, linking it with the URL.
+
+*NOTE: Any text comments made in the document itself without a [Github issue](https://github.com/imfug/002-pkg-and-delivery/issues) link risk been overlooked.*
+
+## Communication
+All communication to the group as a whole pertaining to the progress herein will be done so via the designated IMF UG reflector: [supplementals@imfug.com](mailto:supplementals@imfug.com).
+There will be NO communication to the main reflector unless directed to do so by the UG chair. If you are interested in this work, [subscribe](http://imfug.com/mailman/listinfo/supplementals) to the the [mailing list here](http://imfug.com/mailman/listinfo/supplementals).
```

Comment:
We should mention that belonging to the mailing list requires membership to the UG, and provide a link to http://www.imfug.com/join/.